server/config/master.key

```diff
@@ -0,0 +1 @@
+fe4605dda57829ddd78fef2e6f103679
```

Comment:
this file should not be added as your credentials looks different from setup of other developers